package-lock.json

```diff
@@ -88,7 +89,7 @@
         "start-server-and-test": "^1.11.4"
       },
       "engines": {
-        "node": "^20",
+        "node": "20.9.0",
```

Comment:
This doesn't seem intentional?

Reply:
Must have arrived via a rebase and has now been disappeared by a rebase.

Reply:
Hmm. The current `package.json` engine is `"node": "20.9.0"` (due to #814) however this didn't get added to `package-lock.json` as part of 2d39dca12901bd3c1b46a33b2a0032c215ae21e5. So it's not intentional - it arrived as part of `npm i @aws-sdk/client-sts` - but it is now correct.

Reply:
I should get in the habit of reading `package-lock.json` more. In the old days it was very chaotic, but it seems to be a little more stable with newer node versions.

Reply:
Yeah, it's pretty stable and diffs tend to be understandable, esp. with Git's [moved-lines highlighting](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---color-movedltmodegt) enabled. I like to read it since it's the actual things we'll get installed.